doc/source/architecture.rst

```diff
@@ -33,8 +33,8 @@ How to set up an aggregator
 
 1. Install Libreant. Follow the instructions on :ref:`dev-installation`.
 
-2. Launch Libreant setting the ``LIBREANT_AGHERANT_DESCRIPTIONS`` environment variable.
-   Its value should be a list of URLs encoded in JSON. Each URL represents the Opensearch 
+2. Launch Libreant setting the ``AGHERANT_DESCRIPTIONS`` configuration parameters.
```

Comment:
On 06/28/2015 09:33 PM, BoySka wrote:
> In doc/source/architecture.rst
> <https://github.com/insomnia-lab/libreant/pull/171#discussion_r33426029>:
> 
>> @@ -33,8 +33,8 @@ How to set up an aggregator
>>  
>>  1. Install Libreant. Follow the instructions on :ref:`dev-installation`.
>>  
>> -2. Launch Libreant setting the ``LIBREANT_AGHERANT_DESCRIPTIONS`` environment variable.
>> -   Its value should be a list of URLs encoded in JSON. Each URL represents the Opensearch 
>> +2. Launch Libreant setting the ``AGHERANT_DESCRIPTIONS`` configuration parameters.
> 
> you really mean there is no prefix anymore on environment variables? I
> think that this is bad.
> environment variables are naturally "inherited" from processes, so
> prefix makes sense to avoid that setting an option for one process will
> also override something ele.

No I didn't mean that. I removed prefix from env variable name but I
also changed "environment variable" to "configuration paramenter". Cli
revolution has changed configuration handling a bit, we now have
multiple way of passing configuration parameters:
https://github.com/insomnia-lab/libreant/pull/162

I think that in docs it would be better to not suggest a specific method
but only provide the configuration parameter name and then provide a [
see_also how-to-configure ].

> 
> So if some program that we are not aware of uses an environment
> variables called, for example, |SYNC|, it will set such an option for
> libreant, too. While some conflicts are "obvious" (ie |PATH|) some other
> option might not be that obvious. For example we have |PORT|, which
> could be set for something else, too.
>
This is not our case, we use prefix in envvars for obvious reason.

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/insomnia-lab/libreant/pull/171/files#r33426029>.
> 

-- 
tommy.ael
gpg key: 0xEBD8B3682C4C4645